src/elements/stepper/step-label/step-label.component.ts

```diff
@@ -57,7 +52,11 @@ class SbbStepLabelElement extends SbbIconNameMixin(SbbDisabledMixin(SbbButtonBas
   public override connectedCallback(): void {
     super.connectedCallback();
     this.id ||= `sbb-step-label-${nextId++}`;
+    this.slot ||= 'step-label';
     this.internals.ariaSelected = 'false';
+    if (this.tabIndex < 0) {
```

Comment:
Could you write a comment what happens here? It doesn't look like the same behavior as before

Reply:
This was a mistake. Changed to direct assignment.